_posts/2015-05-30-hot-planet-astrobiology.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Global warming may be key to finding E.T.
+author: neuroamanda
+---
+
+_Exoplanets that are too hot may be a sign of alien life_
+
+A new research paper suggests that scanning for waste heat radiating off of Earth-like planets may be important in the search for extraterrestrial intelligent life.
+Like the radio signals emitted by Earth, heat may serve as an interstellar smoke signal indicating energy production and consumption.
+While this not a new idea, it may be a consideration for how we build future large telescopes.
+
+With nearly 2000 exoplanets discovered so far &mdash; of which 40-odd are potentially habitable &mdash; Earth may not be as special in its ability to harbor life as once thought.
+One signal that we have been listening for in the search for extraterrestrial intelligence is radio waves, the dominant output of our rapidly industrialized Earth civilization over the past 100 years or so.
+But even these signals are becoming silenced with the end of terrestrial broadcasting; if the same thing happened to ET, what are some other large-scale signs that could indicate the presence of intelligent life?
+The trend on this planet is towards ever greater energy consumption, especially as the economy increasingly depends on storing and manipulating information.
+The more advanced a civilization is, the argument goes, the more power it generates, and the more it will move towards harnessing the power of its solar system, perhaps culminating in a [Dyson sphere](http://en.wikipedia.org/wiki/Dyson_sphere).
+The thermal radiation of the planet may then be an unintentional calling card, a biomarker of extraterrestrial life.
+In particular, the way heat is distributed as islands in space and time on a planet's surface &mdash; think of the bright spots of illuminated cities on Earth at night &mdash; may be a signature of Earth-like civilizations.
+
+![Kepler-16b, an exoplanet orbiting a pair of stars]({{ site.url }}/images/2015-05-30-Kepler_16b.jpg)
+
+> _Kepler-16b, an exoplanet orbiting a pair of stars. Modified from original [image](http://planetquest.jpl.nasa.gov/system/secondary_files/binaries/988/original/Kepler_16b_39x27.jpg) / Courtesy NASA/JPL-Caltech / Public Domain_
+
+The researchers modeled what heat signals might come off an exoplanet, using Earth as a basis.
+Weighing the energy inputs (solar and planetary heat sources) and outputs (photosynthesis, biological activity, human/alien technology) for their model planet, they reached the conclusion that as you produce more energy that is returned to the planet as heat, the model planet starts to warm up.
```

Comment:
"Technology" implies some intentional work behind it, human or alien. You may want to drop the qualifier since it doesn't really add value and can be a tad confusing in the context of this par.